websocketClient.go

```diff
@@ -28,13 +28,15 @@ type ResponseAnswerStruct struct {
 func websocketClient() {
 
 	var clientUrl = Config.GetWSClientUrl()
+    log.Printf("signaling ws: %v", clientUrl)
```

Comment:
wrong indent (space vs tab)
in other places as well

Reply:
@yurp hey next time just merge the PR since its an Embeded feature